.github/workflows/pr_build.yml

```diff
@@ -36,6 +36,25 @@ jobs:
           pip install pytest
           pytest contract-tests/tests
 
+  build-lambda:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo @ SHA - ${{ github.sha }}
+        uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        if: ${{ matrix.language == 'python' }}
```

Comment:
it seems we don't have `matrix.language` defined for this job

Reply:
tox will cover all the python versions